src/vs/editor/browser/services/codeEditorServiceImpl.ts

```diff
@@ -310,7 +310,7 @@ class DecorationRenderHelper {
 				cssTextArr.push(strings.format(this._CSS_MAP.contentIconPath, opts.contentIconPath.toString(true).replace(/'/g, '%27')));
```

Comment:
Ok makes sense. But the best thing would be to provide a seperate PR that only fixes this issue.
That PR I will then assign to the editor people. As the description just copy this comment.